src/widgets/geo-search/geo-search.ts

```diff
@@ -269,7 +269,7 @@ const geoSearch: GeoSearchWidget = (widgetParams) => {
         templateKey: 'HTMLMarker',
         templates,
         data: item,
-      }),
+      }) as string,
```

Comment:
For now, `geoSearch` templates will only support strings. We'll assess separately how to bring `html` to this widget.

Reply:
don't we need to warn or error when a jsx is returned then?

Reply:
Yes, I planned on tackling it separately.

Reply:
What's different in `geoSearch()` to not support JSX?

Reply:
`geoSearch` is implemented in #5092.